### PR TITLE
[Cloudtrust-1617] update account

### PR DESCRIPTION
**Pull request by harture**

### CODE REVIEW

File: users.go [hunk 1]
```diff
@@ -145,3 +147,16 @@ func (c *Client) UpdatePassword(accessToken, realm, currentPassword, newPassword
 	var m = map[string]string{"currentPassword": currentPassword, "newPassword": newPassword, "confirmation": confirmPassword}
 	return c.post(accessToken, nil, url.Path(accountPasswordPath), url.Param("realm", realm), body.JSON(m))
 }
+
+// GetAccount updates the user's informations
```
Comment (bsoniam) [inline users.go]: // GetAccount returns/provides (?) the user's information. (no s)

File: users.go [hunk 2]
```diff
@@ -145,3 +147,16 @@ func (c *Client) UpdatePassword(accessToken, realm, currentPassword, newPassword
...
+func (c *Client) GetAccount(accessToken string, realm string) (UserRepresentation, error) {
+	var resp = UserRepresentation{}
+	var err = c.get(accessToken, &resp, url.Path(accountPath), url.Param("realm", realm), headers.Set("Accept", "application/json"))
+	return resp, err
+}
+
+// UpdateAccount updates the user's informations
```
Comment (bsoniam) [inline users.go]: information

File: users.go (hunk 1 above)
Comment (harture) [reply users.go]: thks

File: users.go (hunk 2 above)
Comment (harture) [reply users.go]: thks

**Pull request merged**